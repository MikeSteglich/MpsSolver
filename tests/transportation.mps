* CMPL - Free-MPS - Export
NAME transportation
* OBJNAME costs
* OBJSENSE MIN
ROWS
 N costs
 E line_2
 E line_3
 E line_4
 L line_5
 L line_6
 L line_7
 L line_8
COLUMNS
 x[1,1] costs 3 line_2 1
 x[1,1] line_5 1
 x[1,2] costs 2 line_2 1
 x[1,2] line_6 1
 x[1,4] costs 6 line_2 1
 x[1,4] line_8 1
 x[2,2] costs 5 line_3 1
 x[2,2] line_6 1
 x[2,3] costs 2 line_3 1
 x[2,3] line_7 1
 x[2,4] costs 3 line_3 1
 x[2,4] line_8 1
 x[3,1] costs 2 line_4 1
 x[3,1] line_5 1
 x[3,3] costs 4 line_4 1
 x[3,3] line_7 1
RHS
 RHS line_2 5000 line_3 6000
 RHS line_4 2500 line_5 6000
 RHS line_6 4000 line_7 2000
 RHS line_8 1500
BOUNDS
 PL BOUND x[1,1]
 PL BOUND x[1,2]
 PL BOUND x[1,4]
 PL BOUND x[2,2]
 PL BOUND x[2,3]
 PL BOUND x[2,4]
 PL BOUND x[3,1]
 PL BOUND x[3,3]
ENDATA
