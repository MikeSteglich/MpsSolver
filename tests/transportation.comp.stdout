---------------------------------------------------------------------------------------------------------
Problem              transportation
Nr. of variables     8
Nr. of constraints   7
Objective name       costs
Solver name          CBC
Display variables    nonzero variables (all)
Display constraints  nonzero constraints (all)
---------------------------------------------------------------------------------------------------------

Objective status     optimal
Objective value      39500 (MIN!)

Variables 
Name                 Type            Activity         Lower bound         Upper bound            Marginal
---------------------------------------------------------------------------------------------------------
x[1,1]                  C                3500                   0                 inf                   0
x[1,2]                  C                1500                   0                 inf                   0
x[2,2]                  C                2500                   0                 inf                   0
x[2,3]                  C                2000                   0                 inf                   0
x[2,4]                  C                1500                   0                 inf                   0
x[3,1]                  C                2500                   0                 inf                   0
---------------------------------------------------------------------------------------------------------

Constraints 
Name                 Type            Activity         Lower bound         Upper bound            Marginal
---------------------------------------------------------------------------------------------------------
line_2                  E                5000                5000                5000                   3
line_3                  E                6000                6000                6000                   6
line_4                  E                2500                2500                2500                   2
line_5                  L                6000                -inf                6000                   0
line_6                  L                4000                -inf                4000                  -1
line_7                  L                2000                -inf                2000                  -4
line_8                  L                1500                -inf                1500                  -3
---------------------------------------------------------------------------------------------------------
